### DISCUSSION TITLE
Supressing informative console log

### DISCUSSION THREAD

**Discussion by jensdah** [sachinchoolur/lightGallery#1284] [Q&A]
Is it possible to supress the console log message below?

"If you want srcset or picture tag to be supported for older browser please include picturefil javascript library in your document."

**Comment by sachinchoolur** [comment]
Yes, You can disable it by passing the option `supportLegacyBrowser`

Docs - https://www.lightgalleryjs.com/docs/settings/#supportLegacyBrowser
